### PR TITLE
Ubuntu core takeover 4341

### PR DESCRIPTION
**Pull request by inuwa**
## Done

Ubuntu core takeover has been created to match [copy doc](https://docs.google.com/document/d/1Wdz3mW5F2Ul64sO2dJBcaELFMN--ewC0fhhJDWxf4XU/edit#) and [design spec](https://github.com/ubuntudesign/www.ubuntu.com-design/issues/164)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- [List additional steps to QA the new features or prove the bug has been resolved]

## Issue / Card

Fixes https://github.com/canonical-websites/www.ubuntu.com/issues/4341
## Screenshots

image
image

### CODE REVIEW

**Comment by inuwa** [comment]
@anasereijo made the changes😀

**Comment by shivjoshi1996** [comment]
From @pmahnke  - This can be merged into master but taken out of the takeover cycle, and we will add it back in when we are ready to present it. @inuwa @anthonydillon

**Comment by inuwa** [comment]
@anthonydillon I removed Ubuntu Core from circulation. Please could you help QA the branch, Thank you.

**Comment by pmahnke** [comment]
this has a commit on the stats page?

**Comment by inuwa** [comment]
> this has a commit on the stats page?

@pmahnke there was an issue with the maps showing blank tooltips in mobile view. which was fixed along with this. I explained it to @shivjoshi1996 earlier and he said that in the future, it might be better added as a separate issue but may be okay now.

File: templates/takeovers/_ubuntu-core-18-takeover.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<section lang="en" data-lang="{{ locale }}" class="p-strip--image is-dark is-deep u-image-position js-takeover p-takeover-ubuntu-core  {% if not default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
```
Comment (anthonydillon) [inline templates/takeovers/_ubuntu-core-18-takeover.html]: We dont use `data-lang` at all and please remove the `lang` as this takeover is for all

File: templates/takeovers/_ubuntu-core-18-takeover.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  <div class="row u-vertically-center">
```
Comment (anthonydillon) [inline templates/takeovers/_ubuntu-core-18-takeover.html]: Why are you u-vertically-center here?

File: templates/takeovers/_ubuntu-core-18-takeover.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <div class="col-6">
+      <h1 style="font-weight: 100;">Introducing Ubuntu <br /> Core 18</h1>
+      <h4 class="u-sv3">Learn what’s new in Ubuntu Core - the secure, lightweight and robust operating system for the internet of things.</h4>
+      <p class="u-align--center u-hide--medium u-hide--large">
+          <img src="{{ ASSET_SERVER_URL }}635ce124-IOT_core_infographic+light.svg" alt="Ubuntu Core 18"/>
+      </p>
+      <p class="u-no-margin--top u-no-margin--bottom"><a href="https://www.brighttalk.com/webcast/6793/339167?utm_source=Takeover" class="p-button--positive">Register for the webinar</a></p>
+      <p class="u-no-margin--top"><a href="https://www.ubuntu.com/download/iot" class="p-link--inverted">Download Ubuntu Core 18 now&nbsp;&rsaquo;</a></p>
```
Comment (anthonydillon) [inline templates/takeovers/_ubuntu-core-18-takeover.html]: Relative link please

File: templates/takeovers/_ubuntu-core-18-takeover.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+    <div class="col-6 u-hide--small">
+        <img src="{{ ASSET_SERVER_URL }}635ce124-IOT_core_infographic+light.svg" alt="K8 Webinar" />
```
Comment (anthonydillon) [inline templates/takeovers/_ubuntu-core-18-takeover.html]: Over indented, should be 2 spaces

File: templates/takeovers/_ubuntu-core-18-takeover.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<section lang="en" data-lang="{{ locale }}" class="p-strip--image is-dark is-deep u-image-position js-takeover p-takeover-ubuntu-core  {% if not default %}u-hide{% endif %}" {% if default %}data-default="true"{% endif %}>
+  <div class="row u-vertically-center">
+    <div class="col-6">
+      <h1 style="font-weight: 100;">Introducing Ubuntu <br /> Core 18</h1>
+      <h4 class="u-sv3">Learn what’s new in Ubuntu Core - the secure, lightweight and robust operating system for the internet of things.</h4>
+      <p class="u-align--center u-hide--medium u-hide--large">
+          <img src="{{ ASSET_SERVER_URL }}635ce124-IOT_core_infographic+light.svg" alt="Ubuntu Core 18"/>
```
Comment (anthonydillon) [inline templates/takeovers/_ubuntu-core-18-takeover.html]: Over indented, should be 2 spaces

File: templates/takeovers/_ubuntu-core-18-takeover.html (hunk 1 above)
Comment (anthonydillon) [inline templates/takeovers/_ubuntu-core-18-takeover.html]: Please remove default template stuff please

File: static/js/desktopStatistics.js [hunk 6]
```diff
@@ -695,6 +695,9 @@ function createMap(selector, options, mapData, countryNamesAndIds) {
             .style("position", "absolute")
             .style("opacity", "1");
           var noOfUsers =  (countryStat.users.toFixed(1) < 1) ? "< 0.1" : countryStat.users.toFixed(1);
+          
+          // unhiding the tooltip message tom prevent tooltip from showing when no country has been hovered over
+          tooltipMessage.style("display", "block");
```
Comment (anthonydillon) [inline static/js/desktopStatistics.js]: Was this just a drive by or should this have been in its own commit?

**Pull request merged**